portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Andrea Ramírez</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="topNavigationBar">
+        <a class="active" href="./index.html">Andrea Ramírez Portfolio</a>
+        <a href="./gallery.html">gallery</a>
+        <a href="./projects.html">projects</a>
+    </div>
+    <h1>Andrea's Projects 🛠️</h1>
+    <h2 style="float: left;"> Punto Blanco Mobile App </h2>
+    <p style="float: left;">Developed a mobile app in Swift for Punto Blanco, Meditation and Wellness Center at ITESM, in a group of 3. <br>
+       App helps users follow guided meditations, record their performance and visualize their own data over a period of time <br>
+      In charge of implementation and maintenance of Firebase database</p>
+    <h2 style="float: left;"> Survive the Outbreak Videogame</h2>
+    <p style="float: left;">Created a three-level videogame in Java in a group of four. Similar to Pacman, designed to teach children healthy habits during quarantine. <br>
+        Led design of user interface, the theme and worked with Photoshop and Illustrator.</p>
+    <h2 style="float: left;">Database for Coronavirus patients </h2>
```

Comment:
In general it is a good idea to add all style elements like float:left in the css files

Reply:
I included those lines of code into a <div> tag with an id = "projects", and added the "float: left"  in the css file under that id. But it keeps going back to the center of the page when I run it
I was thinking I could create a different .css file, but I don't think that's the best option

Reply:
*into a div tag

Reply:
If you need it to left align add something like this:

.projects h2{
    text-align: left;
    margin: 1em;
}

.projects p{
   text-align: left;
    margin: 1em;
}

Basically currently your style is being overwritten by the default p & h2 properties. So basically here we need to add properties that are specific to your block.

Reply:
Thank you Sakshi! It did work :)
I was not noticing it was being overwritten